docs/CONTRIBUTING.md

```diff
@@ -14,17 +14,31 @@ If you are a *git ninja*, we feel happy and sad for you at the same time. Your s
+`[Python][xbmcgui] Fix x problem...` or `Python/xbmcgui: Fix x problem...`<br/>
+instead of something like `[README.md] Update`, which is **not** descriptive enough.<br/>
+Also, adding a short description to the *commit* (under the title) is a good thing to consider to give a better explanation of the commit changes.<br/>
+The *pull request* description should only contain information relevant to the change.
+* **Draft PR**. If your work requires further development, and you wish to ask the opinion of other developers on how to proceed, you can publish your PR as Draft.
+
+## Pull request merge guidelines
+* **First rule, be patient, be kind, rewind**. We're all volunteers with busy lives but usually very responsive in cataloging *pull requests* and pinging the relevant maintainer(s). Although not valid for the whole codebase, specific components of Kodi have a set of [maintainers/code owners](https://github.com/xbmc/xbmc/blob/master/.github/CODEOWNERS) whose review will be automatically requested when you open a pull request.
+* **Jenkins code formatting requests**. Always apply code style formatting requests from Jenkins. If not, please provide a valid reason why you do not apply them.
+* **Jenkins CI builds**. It is mandatory to have green light on all Jenkins building platforms. The "No-Jenkins" label can be set by maintainers to avoid having the CI building the PR code (and wasting build resources) in cases the contribution is still work in progress or if known in advance it won't build.
+* **Reviewing time**. Be patient, in general terms we speak of weeks not days, the active team is small and that means things can stack up, even as we are all doing our best. If your PR has been standing for too long without a review, feel free to let us know.
+* **Reviewing process**. At least one member's approval is required to have the green light to merge. Please take into account that expertise is spread thinly across areas of Kodi. This makes getting reviews of some changes even harder. Kodi is tangled and what looks perfect to you may have unwanted side effects, so the approval by several members sometime is required. In any case, this is always helpful, in particular for huge additions/refactors that may cause regressions. Therefore, it is possible that your code will require more love, i.e., more changes before its final version may be required, so arm yourself with relaxing herbal teas.
+* **Test builds**. When reasonable, providing test builds to test your changes can speed up the review process, especially if you consider that some users or team members cannot build Kodi for testing. If you are a member of Team Kodi, we remind you that there are Jenkins helpers that can build and upload onto the mirrors where users can access them to test it. If you are not part of the team, you can make a request for test builds.
```

Comment:
... that can build and upload code onto the mirrors where users can access it for test purposes.